Proton/Sources/Core/RichTextEditorContext.swift

```diff
@@ -30,9 +30,9 @@ class RichTextEditorContext: RichTextViewContext {
 
         activeTextView = textView as? RichTextView
         guard let richTextView = activeTextView else { return }
-        activeTextView?.richTextViewDelegate?.richTextView(richTextView, didReceiveFocusAt: textView.selectedRange)
-
-        let range = textView.selectedRange
+        
+        let range = richTextView.selectedRange
+        richTextView.richTextViewDelegate?.richTextView(richTextView, didReceiveFocusAt: range)
```

Comment:
I changed these lines around to avoid calls to `textView` (needlessly untyped) and `activeTextView` (needlessly optional). Instead, using `richTextView` where we can.